### ISSUE TITLE
bug: `<a class="card">` doesn't get focus outline

### ISSUE THREAD

**Issue by jrunning** [saadeghi/daisyui#4421]
### Reproduction URL (Required)

https://codesandbox.io/p/devbox/5mpjym

### What version of daisyUI are you using?

v5.5.17

### Which browsers are you seeing the problem on?

Chrome, Safari, Firefox

### Describe your issue

When using an `<a>` as a [card](https://daisyui.com/components/card/), the element doesn't receive a focus ring when navigating with the <kbd>Tab</kbd> key. This is an accessibility bug. Links should always get focus rings by default. I've found that adding `outline-2` (e.g.) and `outline-solid` makes the focus ring appear, but this shouldn't be necessary.

This is the markup in question:

```html
<a class="card bg-base-300" href="http://example.com" tabindex="0">
  <div class="card-body">I'm a card.</div>
</a>
```

You can see the behavior (and the workaround) at the reproduction URL. It may be easier to observe the behavior if you click on the button to open the preview in a new tab; it looks like this: <img width="24" height="20" alt="Image" src="https://github.com/user-attachments/assets/5babdef1-627a-4160-9767-a7501400f59f" />.

Thanks for your hard work on daisyUI and for taking a look at this issue.

**Comment by clint-code** [comment]
Hi! I'd like to work on this issue. I'll fix the <a class="card"> accessibility issue. Please assign it to me!

**Comment by pdanpdan** [comment]
@clint-code Can you please first finish https://github.com/saadeghi/daisyui/pull/4460?
